packages/db/prisma/schema.prisma

```diff
@@ -67,22 +69,34 @@ model Session {
 }
 
 model User {
-    id               String          @id @default(cuid())
-    name             String?
-    email            String?         @unique
-    emailVerified    DateTime?
-    image            String?
-    accounts         Account[]
-    sessions         Session[]
-    payments         Invoice[]
-    preference       Preference?
-    membership       Membership      @default(FREE)
-    hasTrialed       Boolean         @default(false)
-    stripeCustomer   StripeCustomer? @relation(fields: [stripeCustomerId], references: [id])
-    stripeCustomerId String?
-
-    createdAt DateTime @default(now())
-    updatedAt DateTime @default(now()) @updatedAt
+    id                       String          @id @default(cuid())
+    name                     String?
+    email                    String?         @unique
+    emailVerified            DateTime?
+    image                    String?
+    sessions                 Session[]
+    preference               Preference?
+    // ================
+    // Stripe stuff
+    // ================
+    membership               Membership      @default(FREE)
+    paidUntil                Int? // in milliseconds
+    stripeCustomerId         String?
+    stripePlanId             String?
+    subscriptionCanceledDate Int?
+    payments                 Invoice[]
+    hasTrialed               Boolean         @default(false)
+    stripeCustomer           StripeCustomer? @relation(fields: [stripeCustomerId], references: [id])
+    // ================
+    // Stripe stuff
+    // ================
+    createdAt                DateTime        @default(now())
+    updatedAt                DateTime        @default(now()) @updatedAt
+    // one user has many accounts
+    accounts                 Account[]       @relation("UserToAccountMany")
+    // one user has one primaryAccount as the organiser of events; optional cause it's set with a custom events.linkAccount hook
+    accountPrimaryId         String?         @unique
+    accountPrimary           Account?        @relation("UserToAccountPrimary", fields: [accountPrimaryId], references: [id])
```

Comment:
primary account now lives as a one-to-one relationship between user<>account

cc: @juliusmarminge @bgavas